packages/devtools_app/lib/src/service/isolate_manager.dart

```diff
@@ -63,15 +63,17 @@ class IsolateManager extends Disposer {
   }
 
   IsolateState? get mainIsolateDebuggerState {
-    return _isolateStates[_mainIsolate.value!];
+    return _mainIsolate.value != null
+        ? _isolateStates[_mainIsolate.value!]
+        : null;
   }
 
   IsolateState? isolateDebuggerState(IsolateRef? isolate) {
-    return _isolateStates[isolate!];
+    return isolate != null ? _isolateStates[isolate] : null;
   }
 
   IsolateState? get selectedIsolateState {
```

Comment:
if this is just returning another getter `mainIsolateDebuggerState`, let's get rid of the selectedIsolateState getter entirely

Reply:
done